src/applications/renderer/src/components/standalone/component.js

```diff
@@ -53,6 +53,7 @@ const Standalone = ({ thumbnail, widgetConfig }) => {
       {isMap && (
         <Suspense>
           <Map
+            layerId={widgetConfig?.paramsConfig?.layer}
```

Comment:
Hi! @pablopareja i added `layerId` to the standalone renderer as-well, as the map legend was not shown on the embed version of the map, to verify, select a map layer and choose renderer. You should see a legend there as-well.